package.json

```diff
@@ -1,15 +1,16 @@
 {
-  "homepage": "https://ahmedadel56.github.io/Calculator",
   "name": "react-calculator",
   "version": "0.1.0",
   "private": true,
+  "homepage": "https://ahmedadel56.github.io/Calculator",
```

Comment:
- [ ] Kindly deploy the actual project online and include the correct livelink in the readme.